runtime/interpreter/value.go

```diff
@@ -795,6 +873,38 @@ func (v CharacterValue) Equal(_ *Interpreter, _ LocationRange, other Value) bool
 	return v.NormalForm() == otherChar.NormalForm()
 }
 
+func (v CharacterValue) Less(interpreter *Interpreter, other ComparableValue, locationRange LocationRange) BoolValue {
+	otherChar, ok := other.(CharacterValue)
+	if !ok {
+		return false
+	}
```

Comment:
Behavior here is inconsistent with `BoolValue` and `IntValue`; if the operands are not both `CharacterValue`s this should produce an `InvalidOperandsError`. Same goes for all the other comparison functions on this type.

Reply:
Thanks, I've updated the code to `panic(errors.NewUnreachableError())`.